ArjunaJTS/trailmap/build.xml.deprecated

```diff
@@ -38,7 +38,8 @@
     <property name="jbosstshome" value=".."/>
```

Comment:
How come this file was deprecated?

Reply:
because there is all build done by maven (see pom.xml). the ant build and the maven build does the same (from my experience)

Reply:
OK, thanks